extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubePlaylistExtractor.java

```diff
@@ -184,18 +185,19 @@ public String getSubChannelAvatarUrl() {
         final StreamInfoItemsCollector collector = new StreamInfoItemsCollector(getServiceId());
         Page nextPage = null;
 
-        final JsonArray contents = initialData.getObject("contents").getObject("twoColumnBrowseResultsRenderer")
-                .getArray("tabs").getObject(0).getObject("tabRenderer").getObject("content")
-                .getObject("sectionListRenderer").getArray("contents").getObject(0)
-                .getObject("itemSectionRenderer").getArray("contents");
+        final JsonArray contents = initialData.getObject("contents")
+                .getObject("twoColumnBrowseResultsRenderer").getArray("tabs").getObject(0)
+                .getObject("tabRenderer").getObject("content").getObject("sectionListRenderer")
+                .getArray("contents").getObject(0).getObject("itemSectionRenderer")
+                .getArray("contents");
 
         if (contents.getObject(0).has("playlistSegmentRenderer")) {
             for (final Object segment : contents) {
-                if (((JsonObject) segment).getObject("playlistSegmentRenderer").has("trailer")) {
-                    collectTrailerFrom(collector, ((JsonObject) segment));
-                } else if (((JsonObject) segment).getObject("playlistSegmentRenderer").has("videoList")) {
```

Comment:
Is there no `trailer` anymore?

Reply:
I ran the learning playlist test and tried in the app this playlist and it works, so I don't know if YouTube doesn't use it anymore or not.

Reply:
Ok, then no need to change